docs/source/library/EccFramework.rst

```diff
@@ -0,0 +1,7 @@
+apply_ecc
+========================
```

Comment:
This headline is way too long. You could have just called it ECC framework

Reply:
Changed the line

Reply:
I don't see this as resolved

Reply:
I don't understand. The headline `apply_ecc` is the name of the documented function. As is the case for the other documented functions in the documentation (see for example https://raw.githubusercontent.com/cda-tum/qecc/main/docs/source/library/SamplePauliError.rst)

Reply:
It's all about the symbols that declare the headline (`#`, `-`, etc.). The length of these lines has to _exactly_ match the length of the headline text. Sorry for not being clear enough.

Reply:
Oh. I see. Thank you for the explanation. I will fix this.